src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderSamePackageDepth253.java

```diff
@@ -22,7 +22,7 @@
 import java.util.StringTokenizer; //#3
 import java.util.concurrent.*; //#4
 import java.util.concurrent.AbstractExecutorService; //#5
-import java.util.concurrent.locks.LockSupport;  // violation
+import java.util.concurrent.locks.LockSupport;  // violation 'Import statement for .*LockSupport' is in the wrong order. Should be in the .*group, expecting not assigned imports.*'
```

Comment:
Each time you were Line wrap it means line is too long, let's shorten messages.

Reply:
ok. got it. Will shorten it.

Reply:
done